sagemaker_studio_image_build/codebuild.py

```diff
@@ -11,14 +11,17 @@
 
 
 class TempCodeBuildProject:
-    def __init__(self, s3_location, role, repository=None, compute_type=None):
+    def __init__(self, s3_location, role, repository=None, compute_type=None, environment=None):
         self.s3_location = s3_location
         self.role = role
 
         self.session = boto3.session.Session()
         self.domain_id, self.user_profile_name = self._get_studio_metadata()
         self.repo_name = None
         self.compute_type = compute_type or "BUILD_GENERAL1_SMALL"
+        self.environment = environment or "LINUX_CONTAINER"
```

Comment:
This default is being specified in multiple places - in the argument parsing and here. From what I can tell, the `environment` param should always be non-null when we get to this point, so we can remove this `or` check.